### ISSUE TITLE
Suggest next useful metric when adding a new metric 

### ISSUE THREAD

**Issue by Sebastiaan127001** [ICTU/quality-time#3579]
Currently, when adding a new metric, Quality Time automatically 'suggests' the metric 'accessibility violations'.
It would be useful if Quality Time chooses the new metric type based on the following criteria.

- Pick a metric from a list of metrics suitable for this subject type
- Pick a metric that is not already used
- (optionally leave out the less obvious metrics like: all the manual ones, arbitrary ones like team happiness, in short use only the ones that can be automated from a tool source)
- when QT knows which sources are available, this can also be taken into account ( see https://github.com/ICTU/quality-time/issues/3127 )

![image](https://user-images.githubusercontent.com/95095453/159780682-902dfadc-96b7-4bb5-8b89-fed98cb961bf.png)

**Comment by Sebastiaan127001** [comment]
relates to https://github.com/ICTU/quality-time/issues/3130
